packages/contracts/test/framework/dao/dao-factory.ts

```diff
@@ -646,15 +639,15 @@ describe('DAOFactory: ', function () {
         EMPTY_DATA
       );
 
-      let DAO_INTERFACE = DAO__factory.createInterface();
-      let PSP_INTERFACE = PluginSetupProcessor__factory.createInterface();
+      const daoInterface = DAO__factory.createInterface();
+      const pspInterface = PluginSetupProcessor__factory.createInterface();
 
       // Prepare actions for apply Installation.
       let applyInstallationActions = [
         {
           to: dao.address,
           value: 0,
-          data: DAO_INTERFACE.encodeFunctionData('grant', [
+          data: DAO__factory.createInterface().encodeFunctionData('grant', [
```

Comment:
use the interface created on line 642

Reply:
Fixed in 2ab0e5ccdb113e25b3aabe2eeca3301faef3f26b